src/plugins/reporter-rhtsupport.c

```diff
@@ -123,33 +123,28 @@ int create_tarball(const char *tempfile, struct dump_dir *dd,
                     basename++;
                 else
                     basename = content;
-                char *xml_name = g_build_filename("content", basename, NULL);
+                g_autofree char *xml_name = g_build_filename("content", basename, NULL);
                 reportfile_add_binding_from_namedfile(file,
                         /*on_disk_filename */ content,
                         /*binding_name     */ name,
                         /*recorded_filename*/ xml_name,
                         /*binary           */ !(value->flags & CD_FLAG_BIGTXT)
                 );
-                free(xml_name);
             }
         }
     }
 
     /* append all files from dump dir */
     dd_init_next_file(dd);
-    char *short_name, *full_name;
+    g_autofree char *short_name, *full_name;
```

Comment:
Just noticed this: `g_autofree` requires that the variable be initialised, e.g. to `NULL`.

Reply:
Interesting, I'd swear I remember fixing this very line, thanks.